src/app/inventory/store/d2-item-factory.ts

```diff
@@ -435,6 +435,10 @@ export function makeItem(
         DestinyClass.Unknown
     : itemDef.classType;
 
+  // process adept items
+  const isAdept =
+    ((currentBucket?.inWeapons || currentBucket?.inPostmaster) && name.includes('Adept')) ?? false;
+
```

Comment:
This logic relies on the user running with the English Bungie definitions, and excludes VoG Timelost and KF Harrowed weapons. Two options:

1. Use our generated list of adept weapon hashes https://github.com/DestinyItemManager/d2-additional-info/blob/master/src/generate-adept-weapon-hashes.ts which is already used in DIM https://github.com/DestinyItemManager/DIM/blob/74724730b6cc70d3c8512d4ab53f64cdbdb6b9b9/src/app/utils/item-utils.ts#L333 -- this wouldn't work for D1 though.
2. Use what Compare uses to strip the localized adept suffix https://github.com/DestinyItemManager/DIM/blob/74724730b6cc70d3c8512d4ab53f64cdbdb6b9b9/src/app/compare/compare-buttons.tsx#L116-L125 -- this needs an addition for (Harrowed), it seems.